pkg/quarterdeck/apikeys_test.go

```diff
@@ -204,6 +202,60 @@ func (s *quarterdeckTestSuite) TestCannotCreateAPIKeyInUnownedProject() {
 	s.CheckError(err, 400, "field restricted for request: org_id")
 }
 
+func (s *quarterdeckTestSuite) TestCannotCreateAPIKeyWithoutPermissions() {
```

Comment:
This test should cover both SC-14746 and SC-15075 from happening at the API level.